classes/Kohana/Log.php

```diff
@@ -107,6 +122,29 @@ public function detach(Log_Writer $writer)
 	}
 
 	/**
+	 * TRUE if Log is set to write immediately, FALSE otherwise
+	 * 
+	 * @return boolean
+	 */
+	public function get_immediate_write()
+	{
+		return $this->write_immediately;
+	}
+
+	/**
+	 * Set/unset immediate writing
+	 * 
+	 * @param boolean $write_immediately
+	 * @return Log
+	 */
+	public function set_immediate_write($write_immediately = TRUE)
```

Comment:
Does this need the default `TRUE` in the parameter? I would say that explicitly setting this would make this function more obvious and gives it less redundancy.